docs/overview/cmake-can-not.rst

```diff
@@ -78,36 +78,36 @@ Correct workflow:
 .. image:: /overview/images/good-workflow.png
   :align: center
 
-That said, it's not enough to know that if you want to add new library to your
+It's not enough to know that if you want to add a new library to your
 ``Visual Studio`` solution you can do:
 
 * :menuselection:`Add --> New Project ... --> Visual C++ --> Static Library`
 
-You have to know that this must be done by adding new
+You have to know that this must instead be done by adding a new
 ``add_library`` command to ``CMakeLists.txt``.
 
 Incomplete functionality coverage
 =================================
 
 There are some missing features in :ref:`CMake <CMake>`. Mapping of
 CMake functionality <-> :ref:`native build tool <Native build tool>` functionality
-is not always bijective. Often this can be workarounded by generating different
+is not always bijective. Often this can be worked around by generating different
 native tool files from the same CMake code. For example it's possible using
 autotools create two versions of library
 (:doc:`shared + static </tutorials/libraries/static-shared>`) by one run.
-However this may affect performance or even not possible for other platforms
+However this may affect performance, or be outright impossible for other platforms
 (e.g. on Windows). With :ref:`CMake <CMake>` you can generate two versions of
-project from one ``CMakeLists.txt`` file: one for shared and one for static
-variant, effectively running generate/build twice.
+project from one ``CMakeLists.txt`` file: one each for shared and static
+variants, effectively running generate/build twice.
 
-With ``Visual Studio`` you can have two variants x86 and x64 in one solution
+With ``Visual Studio`` you can have two variants, x86 and x64, in one solution
 file. With :ref:`CMake <CMake>` you have to generate project twice:
 once with ``Visual Studio`` generator and one more time with ``Visual Studio Win64``
 generator.
 
-Same with ``Xcode``. In general :ref:`CMake <CMake>` can't mix two different
-toolchains (at least for now) so it's not possible to generate ``Xcode``
-project with ``iOS`` and ``OSX`` targets - again, just generate code for each
+Similarly with ``Xcode``. In general :ref:`CMake <CMake>` can't mix two different
+toolchains (at least for now) so it's not possible to generate an ``Xcode``
+project with ``iOS`` and ``OSX`` targets &mdash again, just generate code for each
```

Comment:
`&mdash`? Encoding issue?

Reply:
Ah, old habit. I'll replace with '—' (should be this instead of '-')